src/libs/antares/study/scenario-builder/scBuilderUtils.h

```diff
@@ -39,6 +39,11 @@ namespace Data
 {
 namespace ScenarioBuilder
 {
+	const double NOT_A_DOUBLE = 12345;
```

Comment:
We could use [`std::nan`](https://en.cppreference.com/w/cpp/numeric/math/nan) as an undefined value instead of a antares custom on ?